readme.md

```diff
@@ -1,5 +1,4 @@
 # conf [![Build Status](https://travis-ci.org/sindresorhus/conf.svg?branch=master)](https://travis-ci.org/sindresorhus/conf)
-
```

Comment:
Can you keep this line?

Reply:
Yes, of course. I have no idea why this got deleted 😕